core/crypto/cbc-mac.c

```diff
@@ -77,14 +80,37 @@ static TEE_Result crypto_cbc_mac_update(struct crypto_mac_ctx *ctx,
 	}
 
 	while (len >= mc->block_len) {
+		size_t nblocks = MIN(len / mc->block_len,
+				     (size_t)CFG_CRYPTO_CBC_MAC_BUNDLE_BLOCKS);
+
+		if (nblocks > 1) {
+			out_len = nblocks * mc->block_len;
+			if (!out_tmp) {
+				out_tmp = malloc(out_len);
+				if (!out_tmp) {
```

Comment:
I'd rather fall back to process one block at a time than aborting here.

Reply:
I agree, updated.